tools/webpack/interactivity.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
 	entry: {
 		index: './packages/interactivity',
 		router: './packages/interactivity-router',
+		'full-csn': './packages/interactivity-router/src/full-csn.js',
```

Comment:
Let's avoid using acronyms like `csn` in the code that aren't easy to unpack when you don't have the full context. A quick Google search doesn't show anything related for [CSN](https://acronyms.thefreedictionary.com/CSN).

Reply:
In [this commit](https://github.com/WordPress/gutenberg/pull/59707/commits/cd7a112cf072e2ce3d49c2ab912cbdc19eac77eb), I renamed everything to use "Full page client-side navigation" and "full page router". I hope that with that, summed to the new explanation in the opening comment, it's enough for the experiment. Let me know if you feel it isn't clear yet.